snippets/debugging-tips.md

```diff
@@ -0,0 +1,3 @@
+# The 16 JavaScript debugging tips you probably didn’t know
+
+- [Click here to read the full text](https://raygun.com/learn/javascript-debugging-tips)
```

Comment:
Hey, @Darin-Hamouda. Try to add actual information instead of links. 
You could ask yourself what were the most useful tips of this article and describe them in your own words.

Furthermore, you branched from you vocabulary branch. So the changes of that branch now show up in this PR. This is bad practice! Always check out `main` and pull before starting work on a new branch.

To fix it, you should make a new branch from main and add your changes there (since you didn't add much, this will be easiest). I'm closing this PR since you're going to have to start a new one.